src/openrct2/interface/Viewport.cpp

```diff
@@ -1592,13 +1592,13 @@ static bool IsSpriteInteractedWithPaletteSet(
         {
             // TODO: SAR in dpi done with `>> 1`, in coordinates with `/ 2`
             DrawPixelInfo zoomed_dpi = {
-                /* .bits = */ dpi.bits,
-                /* .x = */ dpi.x >> 1,
-                /* .y = */ dpi.y >> 1,
-                /* .height = */ dpi.height,
-                /* .width = */ dpi.width,
-                /* .pitch = */ dpi.pitch,
-                /* .zoom_level = */ dpi.zoom_level - 1,
+                .bits = dpi.bits,
+                .x = dpi.x >> 1,
+                .y = dpi.y >> 1,
+                .width = dpi.width,
```

Comment:
That’s interesting, was this switching height and width around until now?

Reply:
It was, yeah. Compilers immediately complained that the DrawPixelInfo order wasn't adhered to as soon as the names were enabled. Funny how that works. An argument for stronger typing, really.